### ISSUE TITLE
DenseVnet Dice loss remains and does not fall

### ISSUE THREAD

**Issue by kye9216789** [NifTK/NiftyNet#296]
Hi
Thank you to open your novel framework.
I used 3D DenseVnet to predict CT image label.
And before use my data, I tested your organ model and data.
But the **Dice-hinge loss remains** over 0.22
After I changed data to my one, also the loss remains specific value, near 0.5
I used only one data to overfit the network.
network is same with you, and I did not modified structure.

I used every parameter same with you.

I tested several time, and trained 1000 epochs but it remains as same.
Can you tell me why loss remains?

![overfit_test](https://user-images.githubusercontent.com/21051721/49637329-70b06d80-fa48-11e8-8109-ec6527ff07a2.png)

**Comment by whikwon** [comment]
Same issue occurs

**Comment by FarheenRamzan** [comment]
Same issue.

**Comment by Farihaa** [comment]
Same issue.

**Comment by KathrynSch** [comment]
Same issue

**Comment by ericspod** [comment]
This discussion on #208 may be relevant to your issues.

**Comment by anmol4210** [comment]
Hi I was also facing the same issue. But when I changed the loss function from Dice to **GDSC** I was able to train my network and loss also started decreasing.
